bands/templates/band_admin/band_info.html

```diff
@@ -11,183 +11,184 @@
 
 <h2>Band Info for <a href="../bands/index?id={{ band.id }}">{{ band.group.name }}</a></h2>
 
+<br/> <br/>
 <form class="form-horizontal" role="form" method="post" action="band_info">
     {% csrf_token %}
     <input type="hidden" name="id" value="{{ band.id }}" />
 
-    {% if band.panel_name %}
-        <div class="row">
-            <label class="col-sm-2 control-label">Panel Name:</label>
-            <div class="col-sm-6">
-                {{ band.panel_name }}
-            </div>
+    {% if band.panel.name %}
+    <div class="row">
+        <label class="col-sm-2 control-label">Panel Name:</label>
+        <div class="col-sm-6">
+            {{ band.panel.name }}
         </div>
+    </div>
     {% endif %}
 
-    {% if band.panel_length %}
-        <div class="row">
-            <label class="col-sm-2 control-label">Panel Length:</label>
-            <div class="col-sm-6">
-                {{ band.panel_length }}
-            </div>
+    {% if band.panel.length %}
+    <div class="row">
+        <label class="col-sm-2 control-label">Panel Length:</label>
+        <div class="col-sm-6">
+            {{ band.panel.length }}
         </div>
+    </div>
```

Comment:
totally dumb thing, and don't worry about it for this PR, just something to keep in mind later: 

I realize this is might just be your editor doing this automatically as you edit the file, for the sake of being able to spot the important parts of the diff, try to minimize pure whitespace changes here.  when the diff has the whitespace tabs removed, it's a little harder to notice the stuff that actually changed (like ```band.panel_length``` to ```band.panel.length```).   

If possible, save the whitespace edits for another PR built on top of this one that is **just** whitespace edits and nothing else, only so it's easy to review.

----
BUT ALSO: there's a secret github trick that tells it to ignore whitespace changes which is nice for this particular file, it can be viewed here: https://github.com/magfest/bands/pull/17/files?w=1 by adding ```?w=1``` at the end of the URL.  so..... no big deal.